files/ansible-unbound-checkconf.sh

```diff
@@ -26,6 +26,7 @@ config_file_to_validate="${1}"
 version=`unbound-checkconf -h | grep -e '^Version' | cut -f 2 -d ' ' | cut -f 1,2 -d '.'`
 
 case "${version}" in
+    1.6)    option="-f -o directory" ;;
```

Comment:
listing all known versions was rather intentional. but i think the flags we need will not change. it might be a naive assumption but when it is, the test would catch it. let's the command default to `-f -o directory`.

for the record, i do not care older versions. version 1.4 is old enough.